src/map/EsriSceneViewController.js

```diff
@@ -39,10 +39,11 @@
                 });
             };
 
+            // TODO: is this still needed?
```

Comment:
Nah, we can probably just remove it if we're going to use it's contents directly down in `setMap`.